workflow.src

```diff
@@ -0,0 +1,129 @@
+{
+  "title":  "Workflow for editors and other contributors"
+}
+---
+<main>
+      <p>
+        A good practice for GitHub is to use <a href="https://help.github.com/articles/about-pull-requests/">pull requests</a>.
+        It lets you tell others about changes you've pushed to the repository.
```

Comment:
Maybe instead of "It lets you tell others about changes you've pushed to the repository", "It lets you propose changes to others without making changes to the master branch of the repository"?